src/zjs_ble.c

```diff
@@ -492,7 +492,7 @@ static void zjs_ble_connected(struct bt_conn *conn, uint8_t err)
         char addr[BT_ADDR_LE_STR_LEN];
         bt_addr_le_to_str(bt_conn_get_dst(conn), addr, sizeof(addr));
         ble_conn.default_conn = bt_conn_ref(conn);
-        zjs_signal_callback(ble_conn.connected_cb_id, &addr, sizeof(addr));
+        zjs_signal_callback(ble_conn.connected_cb_id, addr, sizeof(addr));
```

Comment:
In C addr and &addr are exactly the same thing for a fixed array like this. I think addr is cleaner because then the syntax the same as for a dynamic array, char *addr;